controllers/datadogagent/component/clusteragent/utils.go

```diff
@@ -69,3 +72,37 @@ func GetClusterAgentService(dda metav1.Object) *corev1.Service {
 
 	return service
 }
+
+// GetMetricsServerServiceName returns the external metrics provider service name
+func GetMetricsServerServiceName(dda metav1.Object) string {
+	return fmt.Sprintf("%s-%s", dda.GetName(), apicommon.DefaultMetricsServerResourceSuffix)
+}
+
+// GetMetricsServerAPIServiceName returns the external metrics provider apiservice name
+func GetMetricsServerAPIServiceName() string {
+	return apicommon.ExternalMetricsAPIServiceName
```

Comment:
thanks - another option is to rm the function and just reference the constant when needed, but i'll leave the decision to you

Reply:
👍 I'll keep it as is for now